### PR TITLE
Dynamically get tooltip content in the render method

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
This allows react-tooltip to change dynamically and properly re-render the tooltip content when changes to its props are made. In my opinion it’s now a better React citizen.

### CODE REVIEW

File: standalone/react-tooltip.js [hunk 1]
```diff
@@ -536,6 +544,7 @@ var ReactTooltip = (0, _staticMethods2.default)(_class = (0, _windowListener2.de
       });
 
       if (globalEventOff) window.removeEventListener(globalEventOff, this.hideTooltip);
+      window.removeEventListener('click', this.onClickOutsideFrozenTooltip);
```
Comment (wbazant) [inline standalone/react-tooltip.js:547]: That looks like something I've originally forgotten thank you!

File: src/index.js [hunk 2]
```diff
@@ -73,14 +72,17 @@ class ReactTooltip extends Component {
       currentTarget: null, // Current target of mouse event
       ariaProps: parseAria(props), // aria- and role attributes
       afterUnfreeze: () => {},
-      isEmptyTip: false,
-      disable: false
+      disable: false,
+      originTooltip: null,
+      isMultiline: false
```
Comment (wbazant) [inline src/index.js:77]: No idea what isMultiline is

Comment (alfonsomunozpomer) [reply src/index.js:77]: From the docs at https://github.com/wwayne/react-tooltip:

|Global|Specific|Type|Values|Description|
|------|--------|-----|--------|----------|
|multiline|data-multiline|Bool|true, false|support &lt;br&gt;, &lt;br /&gt; to make multiline|

File: standalone/react-tooltip.js (hunk 1 above)
Comment (alfonsomunozpomer) [reply standalone/react-tooltip.js:547]: Hmmm... I don’t know why GitHub says this line was added, because it wasn’t! You hadn’t forgotten about it. ;)

**Pull request merged**